backend/app/api/endpoints/dingtalk_docs.py

```
@@ -0,0 +1,131 @@
+        raise HTTPException(
+            status_code=400,
+            detail="DingTalk Docs MCP is not configured. "
+            "Please enable it in Settings > Integrations first.",
+        )
+
+    try:
+        result = await DingTalkDocService.sync_dingtalk_docs(current_user, db)
+        return DingtalkSyncResult(**result)
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)